tpot/base.py

```diff
@@ -275,7 +280,8 @@ def fit(self, features, classes):
         self._start_datetime = datetime.now()
 
         self._toolbox.register('evaluate', self._evaluate_individual, features=features, classes=classes)
-        pop = self._toolbox.population(n=self.population_size)
+        # assign population, self._pop can only be not None if warm_start is enabled
+        pop = self._pop if self._pop else self._toolbox.population(n=self.population_size)
```

Comment:
I know it's less space-efficient, but can you please break this out into a multi-line if/then? Easier to read in the line run.